.travis.yml

```diff
@@ -1,5 +1,5 @@
-sudo: false
 language: generic
+sudo: false
```

Comment:
I mean you did still move it 🤔
Why?

Reply:
Well, I guess one could remove that line, sure.

Reply:
Not remove, just keep it as L1 rather than moving it to L2.

Reply:
Oh, please.